backend/canisters/local_user_index/api/src/lib.rs

```diff
@@ -87,16 +88,24 @@ pub struct UserRegistered {
 
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct BotRegistered {
-    pub user_id: UserId,
+    pub bot_id: UserId,
+    pub owner_id: UserId,
     pub user_principal: Principal,
     pub name: String,
     pub commands: Vec<BotCommandDefinition>,
     pub autonomous_config: Option<AutonomousConfig>,
+    pub initial_install_location: Option<BotInstallationLocation>,
```

Comment:
We don't actually know if the bot is installed in this location.
Maybe something like `permitted_install_location`?